command-line/dot-profile.md

```diff
@@ -12,7 +12,11 @@ Here are links to lessons that should be completed before this lesson:
 
 ### Motivation
 
-Learn about how to customize a computer's environment 
+With knowledge about dot profile, you can customize your computer’s environment.
+
```

Comment:
Can you just list a few things that configuring bash profile would make easier and faster?

Reply:
🎯